response_operations_ui/controllers/case_controller.py

```diff
@@ -107,3 +107,23 @@ def is_allowed_status(status):
 def get_case_group_status_by_collection_exercise(case_groups, collection_exercise_id):
     return next((case_group['caseGroupStatus'] for case_group in case_groups
                  if case_group['collectionExerciseId'] == collection_exercise_id), None)
+
+
+def get_cases_by_sample_unit_id(sample_unit_ids):
+    logger.debug('Retrieving cases for sample unit IDs', sample_unit_ids=sample_unit_ids)
+    url = f'{app.config["CASE_URL"]}/cases/sampleunitids'
+
+    response = requests.get(url=url,
+                            auth=app.config['CASE_AUTH'],
+                            params={'sampleUnitId': sample_unit_ids})
+
+    if response.status_code == 404:
+        logger.error("There were no cases found for sample unit ids", sample_unit_ids)
```

Comment:
I wonder if we'd want to log at error level here. It's quite likely we could have sample file uploaded with addresses in without the cases being created?

Reply:
I agree. Debug/info feels better here.